.codecov.yml

```diff
@@ -13,3 +13,9 @@ coverage:
       default:
         target: 0
         threshold: null
```

Comment:
We may need to change our thresholds to actual numbers now that 4.5.0 has been released. It seems like "null" no longer works.